README.md

```diff
@@ -96,7 +96,7 @@ Every model is written from scratch to maximize performance and remove layers of
 
 | Model | Model size | Author | Reference |
 |----|----|----|----|
-| Llama 3 | 8B, 70B | Meta AI | [Meta AI 2024](https://github.com/meta-llama/llama3) |
+| Llama 3.1 | 8B, 70B, 405B | Meta AI | [Meta AI 2024](https://github.com/meta-llama/llama3) |
```

Comment:
nit: you added only 405B model for 3.1 version.

Reply:
There aren't any other sizes. 8, 70, and 405 are the only ones. https://llama.meta.com/

Reply:
yes, I mean you added only 405B in this PR :)